frontend/src/index.html

```diff
@@ -177,6 +178,7 @@ <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.hig
                     {{#_highlightResult.languages}}
                     <div title="Language spoken in the recording">{{{value}}}</div>
                     {{/_highlightResult.languages}}
+                    <a x-on:click="is_open=!is_open" href="#" class="flex flex-col flex-grow text-center">More...</a>
```

Comment:
- I'm sure this link should be shown only if `topicsReady === true`. 
- Can we make it a span/div instead of a dummy anchor? It may have `cursor-pointer` class to indicate that it's clickable.
- It doesn't seem to have to be a `flex` because it does not have children. Actually I don't think any of the provided classes are needed here.